packages/fmg-payments/src/payment-app.ts

```diff
@@ -0,0 +1,142 @@
+import { Commitment, Channel, CommitmentType } from 'fmg-core';
+import { bigNumberify } from 'ethers/utils';
+
+export interface PaymentCommitmentArgs {
+  libraryAddress: string;
+  channelNonce: number;
+  asAddress: string;
+  bsAddress: string;
+  asBalance: string;
+  bsBalance: string;
+  turnNum: number;
+  commitmentCount: number;
+}
+function fromPaymentCommitmentArgs(paymentArgs: PaymentCommitmentArgs) {
```

Comment:
I'm guessing this should return `Commitment`?

Reply:
The function returns an almost `commitment. Since it doesn't define a `commitmentType` the return type can't be typed as a commitment.